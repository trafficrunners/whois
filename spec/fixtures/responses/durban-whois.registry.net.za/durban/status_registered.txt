Domain Name: wordpress.durban
Domain ID: dom_7G-9999
WHOIS Server: durban-whois1.registry.net.za

Updated Date: 2014-11-11T06:00:03Z
Creation Date: 2014-11-04T06:00:01Z
Registry Expiry Date: 2016-11-04T06:00:01Z
Sponsoring Registrar: MarkMonitor
Sponsoring Registrar IANA ID: 292 
Domain Status: clientDeleteProhibited
Domain Status:clientTransferProhibited
Domain Status:clientUpdateProhibited

Registrant ID: mmr-132163
Registrant Name: DNStination Inc.
Registrant Organization: 
Registrant Street: 425 Market St 5th Floor 
Registrant City: San Francisco
Registrant State/Province: CA
Registrant Postal Code: 94105
Registrant Country: US
Registrant Phone: +1.4155319335
Registrant Phone Ext: 
Registrant Fax: +1.4155319336
Registrant Fax Ext: 
Registrant Email: admin@dnstinations.com

Admin ID: mmr-132163
Admin Name: DNStination Inc.
Admin Organization: 
Admin Street: 425 Market St 5th Floor 
Admin City: San Francisco
Admin State/Province: CA
Admin Postal Code: 94105
Admin Country: US
Admin Phone: +1.4155319335
Admin Phone Ext: 
Admin Fax: +1.4155319336
Admin Fax Ext: 
Admin Email: admin@dnstinations.com

Billing ID: mmr-132163
Billing Name: DNStination Inc.
Billing Organization: 
Billing Street: 425 Market St 5th Floor 
Billing City: San Francisco
Billing State/Province: CA
Billing Postal Code: 94105
Billing Country: US
Billing Phone: +1.4155319335
Billing Phone Ext: 
Billing Fax: +1.4155319336
Billing Fax Ext: 
Billing Email: admin@dnstinations.com

Tech ID: mmr-132163
Tech Name: DNStination Inc.
Tech Organization: 
Tech Street: 425 Market St 5th Floor 
Tech City: San Francisco
Tech State/Province: CA
Tech Postal Code: 94105
Tech Country: US
Tech Phone: +1.4155319335
Tech Phone Ext: 
Tech Fax: +1.4155319336
Tech Fax Ext: 
Tech Email: admin@dnstinations.com


Name Server: ns3.markmonitor.com
Name Server: ns1.markmonitor.com
DNSSEC: unsigned
>>> Last update of WHOIS database: 2015-06-02T09:20:05Z <<<

#    WHOIS lookup made at 2015-06-02T09:20:05Z
# --
# The use of this Whois facility is subject to the following terms and
# conditions. https://registry.net.za/whois_terms
# Copyright (c) ZACR 1995-2015
