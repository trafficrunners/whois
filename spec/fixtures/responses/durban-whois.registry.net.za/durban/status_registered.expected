#domain
  %s == "wordpress.durban"

#domain_id
  %s == "dom_7G-9999"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2014-11-04T06:00:01Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2014-11-11T06:00:03Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2016-11-04T06:00:01Z}


#registrar
  %s %CLASS{registrar}
  %s.id           == "292"
  %s.name         == "MarkMonitor"
  %s.organization == "MarkMonitor"
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == "mmr-132163"
  %s[0].name          == "DNStination Inc."
  %s[0].organization  == ""
  %s[0].address       == "425 Market St 5th Floor"
  %s[0].city          == "San Francisco"
  %s[0].state         == "CA"
  %s[0].zip           == "94105"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4155319335"
  %s[0].fax           == "+1.4155319336"
  %s[0].email         == "admin@dnstinations.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "mmr-132163"
  %s[0].name          == "DNStination Inc."
  %s[0].organization  == ""
  %s[0].address       == "425 Market St 5th Floor"
  %s[0].city          == "San Francisco"
  %s[0].state         == "CA"
  %s[0].zip           == "94105"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4155319335"
  %s[0].fax           == "+1.4155319336"
  %s[0].email         == "admin@dnstinations.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == "mmr-132163"
  %s[0].name          == "DNStination Inc."
  %s[0].organization  == ""
  %s[0].address       == "425 Market St 5th Floor"
  %s[0].city          == "San Francisco"
  %s[0].state         == "CA"
  %s[0].zip           == "94105"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4155319335"
  %s[0].fax           == "+1.4155319336"
  %s[0].email         == "admin@dnstinations.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns3.markmonitor.com"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns1.markmonitor.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
